.gitignore

```diff
@@ -1,4 +1,5 @@
 .*
 __pycache__
 
-!.gitignore
\ No newline at end of file
+!.gitignore
+!.github
```

Comment:
`.github`  shouldn't be ignored

Reply:
That's why it's here. First of all we ignore everything beginning with `.`, so we have to create an exception from the rule - which is done by  the `!` character.